src/crepe/system/ScriptSystem.h

```diff
@@ -33,7 +33,7 @@ class ScriptSystem : public System {
 	 *
 	 * \returns List of active \c Script instances
 	 */
-	std::forward_list<std::reference_wrapper<Script>> get_scripts() const;
+	RefVector<Script> get_scripts() const;
```

Comment:
Je hebt hier forward_list vervangen door vector. Ik neem aan, aangezien dit jouw systeem is, dat vector hier dus ook in orde is.